src/main/scala/firrtl/Emitter.scala

```diff
@@ -1078,6 +1110,16 @@ class VerilogEmitter extends SeqTransform with Emitter {
         emit(Seq("`endif // SYNTHESIS"))
       }
 
+      if (formals.keys.nonEmpty) {
+        emit(Seq("`ifdef FORMAL"))
```

Comment:
do we need this block? do all tools that accept SV accept assert, assume, cover

Reply:
if we keep this block, what should the name be?

Reply:
FIR_VERIFICATION
FIR_FORMAL

Reply:
Agree with @tbd-alcorn about not enclosing it in ifdef FORMAL.   assert, assume, and cover are not restricted to formal, and all simulation tools would benefit from it.